libkbfs/md_journal.go

```diff
@@ -362,12 +362,18 @@ func (j mdJournal) putExtraMetadata(
 		return errors.New("Invalid extra metadata")
 	}
 
+	// TODO: We lose extraV3.wkbNew and extraV3.rkbNew here. Store
```

Comment:
Ah, you're right! The call to `NewExtraMetadataV3` right above should be passing in `true, true`. Not sure how this managed to slip past the MDv3 tests (in my other branch), so will have to investigate. I'll merge once I add a test.